lib/flagship/features.rb

```diff
@@ -0,0 +1,17 @@
+class Flagship::Features < Array
+  def tagged(tags)
+    Flagship::Features.new(select{|feature| tags.any?{|tag, val| feature.tags[tag] == val}})
```

Comment:
Ok how about, I'll rename `tagged` to `tagged_any` but use `tagged_all` as the default `tagged`.